recipes/proxy.rb

```diff
@@ -25,12 +25,16 @@
 
 file "#{node['nginx']['dir']}/conf.d/jenkins.cert" do
   content ssl['client']['cert']
-  mode 0644
+  owner 'root'
```

Comment:
Ill bring this inline with the nginx cookbook, but the root group was something that was already in this cookbook.